### PR TITLE
psh: update psh-applets

### PR DESCRIPTION
**Pull request by mateusz-bloch**
JIRA: CI-426

### CODE REVIEW

File: utils/psh-applets/hm.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# hm
+
```
[the remark this answers is not in the document]
Comment (mateusz-bloch) [reply utils/psh-applets/hm.md:2]: ???

**Review by damianloew — CHANGES_REQUESTED** [review]
Thanks for those descriptions! I think it would be good to leave some note on `psh` page in the `Applets` chapter. Sth like:

`In psh, each command or set of commands is a separate applet. The basic usage of most applets is compatible with POSIX requirements. For example, `ls` works like the `ls` user command on Ubuntu (or other Linux distribution), except for particular arguments that are not supported. The Phoenix shell is constantly under development so the exact behavior may differ a little bit, especially since new capabilities may become available. Here's a list of the provided applets:`

**Review by damianloew — COMMENTED** [review]
@gerard5, @agkaminski could you take a quick look? Maybe you see some issues when reading those descriptions.

**Review by agkaminski — DISMISSED** [review]

File: utils/psh-applets/wget.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# wget
```
Comment (damianloew) [inline utils/psh-applets/wget.md:1]: Please also change the headers of the existing descriptions to lowercase for consistency. Otherwise LGTM

**Review by damianloew — APPROVED** [review]
Thanks, LGTM

**Pull request merged**